### ISSUE TITLE
Semantic Annotations do not accept simple constant sets

### ISSUE THREAD

**Issue by StephanEwen** [stratosphere/stratosphere#832]
The Semantic Annotations on a single input function do not accept simple strings like `"0, 1, 2"`

**Comment by StephanEwen** [comment]
They also do not accept `"0->0,1->1,2->2"`

**Comment by skunert** [comment]
Okay, I am on it.

Also some explanation how it currently works:

**_ConstantFields**_
Here, every input field gets one string. For example this is legit:

```
@ConstantFields({"1->1", "2->2", "3->3"})
```

**_ConstantFieldsExcept**_
In the '''ConstantFieldsExcept''' annotation there is only one String, for example

```
@ConstantFields("1,2,3")
```

I agree with you that this is kind of counterintuitive. I am working on a more streamlined version where you can also use one String for the ConstantFields annotation like your first example.

**Comment by skunert** [comment]
This is handled in PR #840

**Comment by StephanEwen** [comment]
Fixed in #840.
